### DISCUSSION TITLE
Symfony demo "Please install the "intl" extension for full localization capabilities."

### DISCUSSION THREAD

**Discussion by MichaelSchmidt1729** [symfony/symfony#49524] [Q&A]
Hi,

I'm at my wits end.

I'm running PHP 8.1.16, Symfony CLI 5.5, Composer 2.4, I'm on WSL2.

When I install the demo and run it, localhost gives me the error:
The Symfony\Polyfill\Intl\Icu\Locale::setDefault() is not implemented. Please install the "intl" extension for full localization capabilities.
php -m shows: intl is installed. In the php.ini I've commented it in.

But despite symfony -v showing CLI 5.5 and having localhost showing me "my" projects as well as the demo (albeit with the error):
When I type php ./symfony --version I get:

PHP Warning:  PHP Startup: Unable to load dynamic library 'curl' (tried: /usr/lib/php/20210902/curl (/usr/lib/php/20210902/curl: cannot open shared object file: No such file or directory), /usr/lib/php/20210902/curl.so (/usr/lib/php/20210902/curl.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'imap' (tried: /usr/lib/php/20210902/imap (/usr/lib/php/20210902/imap: cannot open shared object file: No such file or directory), /usr/lib/php/20210902/imap.so (/usr/lib/php/20210902/imap.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'mysqli' (tried: /usr/lib/php/20210902/mysqli (/usr/lib/php/20210902/mysqli: cannot open shared object file: No such file or directory), /usr/lib/php/20210902/mysqli.so (/usr/lib/php/20210902/mysqli.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
PHP Warning:  PHP Startup: Unable to load dynamic library 'pdo_mysql' (tried: /usr/lib/php/20210902/pdo_mysql (/usr/lib/php/20210902/pdo_mysql: cannot open shared object file: No such file or directory), /usr/lib/php/20210902/pdo_mysql.so (/usr/lib/php/20210902/pdo_mysql.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
PHP Warning:  Module "intl" is already loaded in Unknown on line 0
Could not open input file: ./symfony

https://github.com/symfony/symfony/discussions/44258 or https://github.com/symfony/symfony/discussions/44258 - which are both about the demo - did not help.

What's wrong?
Whats's the solution?
Please advise!
I'm quite the noob..

**Comment by stof** [comment]
Those `PHP Warning: PHP Startup: Unable to load dynamic library ...` messages indicate that your intsallation of PHP is broken (extensions are not loaded properly). That what you should solve (and it is not related to Symfony itself)
